kafka/1.3/docker-entrypoint.sh

```diff
@@ -39,18 +39,17 @@ if [[ -z "$HOST_NAME" ]]; then
 fi
 
 : ${PORT:=9092}
-: ${ADVERTISED_PORT:=9092}
+
+: ${ADVERTISED_PORT:=${PORT}}
 : ${ADVERTISED_HOST_NAME:=$HOST_NAME}
-: ${LISTENERS:=PLAINTEXT://$HOST_NAME:$PORT}
-: ${ADVERTISED_LISTENERS:=PLAINTEXT://$ADVERTISED_HOST_NAME:$ADVERTISED_PORT}
-export KAFKA_LISTENERS=$LISTENERS
-export KAFKA_ADVERTISED_LISTENERS=$ADVERTISED_LISTENERS
-unset HOST_NAME
-unset PORT
-unset ADVERTISED_PORT
-unset ADVERTISED_HOST_NAME
-unset LISTENERS
-unset ADVERTISED_LISTENERS
+
+: ${KAFKA_ADVERTISED_PORT:=${ADVERTISED_PORT}}
+: ${KAFKA_ADVERTISED_HOST_NAME:=$ADVERTISED_HOST_NAME}
+
+: ${KAFKA_ADVERTISED_LISTENERS:=PLAINTEXT://$KAFKA_ADVERTISED_HOST_NAME:$KAFKA_ADVERTISED_PORT}
```

Comment:
I tested the PR and it seems it is not working as it should. The log contains corretc message but `KAFKA_ADVERTISED_LISTENERS` must be exported otherwise it is not passed to the kafka startup script.
Also there is yet one more compatibility issue - `KAFKA_LISTENERS` should be kept too. It was possible to set PORT and ADVERTISED_PORT for distinct values - same for HOST AND LISTENERS. This no longer available.